packages/runner/src/storage/cache.ts

```diff
@@ -1242,15 +1242,21 @@ export class Replica {
   // Session Identity. Skips if ACL already exists, or if
   // there is no user or space identity.
   private async requestACLInit() {
-    if (!this.spaceIdentity) {
-      return;
-    }
-    const aclKey = this.spaceIdentity.did();
     const userIdentity = (this.remote as any).as as Signer | undefined;
     if (!userIdentity) {
       return;
     }
 
+    // Request ACL in two cases:
+    // 1. When spaceIdentity exists (delegated space)
+    // 2. When user's identity equals the space DID (home space)
+    const isHomeSpace = userIdentity.did() === this.space;
+    if (!this.spaceIdentity && !isHomeSpace) {
+      return;
+    }
+
+    const aclKey = this.spaceIdentity?.did() ?? this.space;
+
```

Comment:
For the home space we shouldn't add anyone. And I think we could actually just leave off the ACL for now, since the default behavior is to treat the space key as owner, which here is the user. @jsantell to confirm.

Reply:
That is true, we can skip the ACL entirely, as the ownership check (in memory/claim.ts) is before the ACL check, so may not need any changes to cache.ts (other than [here](https://github.com/commontoolsinc/labs/pull/2159#discussion_r2558279675) handling opening a space e.g. home space when we have a spaceIdentity for some other string-derived space name)